www/data_module/README.md

```diff
@@ -25,3 +25,17 @@ $ gulp
 $ npm install
 $ karma start
 ```
+
+## How to test within buildbot/www/base ?
+
+```
+$ cp dist/* ../base/libs/buildbot-data/dist/
```

Comment:
my workflow looks something like this:
1. run `gulp prod` in base (dependencies are installed)
2. run `gulp prod` in data
3. create symlink
4. run `gulp dev proxy` in base
5. run `gulp dev` in data

On Mon, Oct 26, 2015, 17:15 Pierre Tardy <notifications@github.com> wrote:

> In www/data_module/README.md
> <https://github.com/buildbot/buildbot/pull/1893#discussion_r43014702>:
>
> > @@ -25,3 +25,17 @@ $ gulp
> >  $ npm install
> >  $ karma start
> >  ```
> > +
> > +## How to test within buildbot/www/base ?
> > +
> > +```
> > +$ cp dist/* ../base/libs/buildbot-data/dist/
>
> are you sure this works?
>
> guanlecoja is supposed to erase the dist directory at startup, which will
> delete your symlink.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/buildbot/buildbot/pull/1893/files#r43014702>.
>